core/src/main/java/com/scalar/db/storage/jdbc/JdbcTableMetadataManager.java

```diff
@@ -105,26 +115,213 @@ public JdbcTableMetadataManager(
     return Optional.of(builder.build());
   }
 
-  private String getSelectColumnsStatement(Optional<String> schemaPrefix, RdbEngine rdbEngine) {
+  private String getSelectColumnsStatement(RdbEngine rdbEngine) {
     return "SELECT "
-        + enclose("column_name", rdbEngine)
+        + enclose(COLUMN_NAME, rdbEngine)
         + ", "
-        + enclose("data_type", rdbEngine)
+        + enclose(DATA_TYPE, rdbEngine)
         + ", "
-        + enclose("key_type", rdbEngine)
+        + enclose(KEY_TYPE, rdbEngine)
         + ", "
-        + enclose("clustering_order", rdbEngine)
+        + enclose(CLUSTERING_ORDER, rdbEngine)
         + ", "
-        + enclose("indexed", rdbEngine)
+        + enclose(INDEXED, rdbEngine)
         + " FROM "
-        + enclosedFullTableName(schemaPrefix.orElse("") + SCHEMA, TABLE, rdbEngine)
+        + enclosedFullTableName(getMetadataSchema(), TABLE, rdbEngine)
         + " WHERE "
-        + enclose("full_table_name", rdbEngine)
+        + enclose(FULL_TABLE_NAME, rdbEngine)
         + " = ? ORDER BY "
-        + enclose("ordinal_position", rdbEngine)
+        + enclose(ORDINAL_POSITION, rdbEngine)
         + " ASC";
   }
 
+  private String getInsertStatement(
+      String schema,
+      String table,
+      String columnName,
+      DataType dataType,
+      @Nullable KeyType keyType,
+      @Nullable Ordering.Order ckOrder,
+      boolean indexed,
+      int ordinalPosition)
+      throws SQLException {
+
+    return String.format(
+        "INSERT INTO %s VALUES ('%s','%s','%s',%s,%s,%s,%d)",
+        enclosedFullTableName(getMetadataSchema(), TABLE, rdbEngine),
+        schemaPrefix.orElse("") + schema + "." + table,
+        columnName,
+        dataType.toString(),
+        keyType != null ? "'" + keyType + "'" : "NULL",
+        ckOrder != null ? "'" + ckOrder + "'" : "NULL",
+        computeBooleanValue(indexed),
+        ordinalPosition);
+  }
+
+  private String computeBooleanValue(boolean value) {
+    switch (rdbEngine) {
+      case ORACLE:
+      case SQL_SERVER:
+        return value ? "1" : "0";
+      default:
+        return value ? "true" : "false";
+    }
+  }
+
+  private String getDeleteStatement(String namespace, String tableName) {
+    String fullTableName = schemaPrefix.orElse("") + namespace + "." + tableName;
```

Comment:
Maybe `schema` is better than `namespace` for consistency. Also `table` is better than `tableName`:
```suggestion
  private String getDeleteStatement(String schema, String table) {
    String fullTableName = schemaPrefix.orElse("") + schema + "." + table;
```

Reply:
Indeed, I will update it.